building.Rmd

```diff
@@ -242,3 +242,7 @@ R --version
     ```sh
     xcode-select --install
     ```
+
+*   libcurl >= 7.28.0 library and headers are required with support for https
+
+    Take anaconda off of your PATH
```

Comment:
```suggestion
    Take anaconda off of your PATH ([SO](https://stackoverflow.com/questions/38690232/installing-r-on-linux-configure-error-libcurl-7-28-0-library-and-headers-a)).
```

Reply:
That question has a lot of other answers aimed at Linux users (including the accepted answer). How about we link to the specific answer [here](https://stackoverflow.com/a/51318770) - though its not particularly detailed.

Reply:
Good idea.